addon/serializers/application.js

```diff
@@ -179,19 +178,23 @@ export default DS.RESTSerializer.extend({
     if ( hasMany && hasMany.get( 'length' ) > 0 ) {
       json[key] = { 'objects': [] };
 
-      if ( options.relation ) {
+      // an array is not a relationship, right?
+      
+      /*if ( options.relation ) {
         json[key].__op = 'AddRelation';
       }
 
       if ( options.array ) {
         json[key].__op = 'AddUnique';
-      }
+      }*/
+      
+      json[key].__op = 'AddRelation';
```

Comment:
I'm not sure where `options.relation` and `options.array` should be coming from. Those properties are undefined in the current master branch